sandbox

```diff
@@ -203,7 +211,13 @@ sandbox () {
   #   dc_pty algod exec //bin/bash
   # This is to ensure compatibility with Windows Msys
   dc () {
-    $DOCKER_COMPOSE -f "$SANDBOX_DIR/docker-compose.yml" "$@"
+    VERBOSE_FLAG=""
+    if [ $VERBOSE_MODE == 1 ]; then
+      # --verbose is universally available since tag https://github.com/docker/compose-cli/releases/tag/v1.0.18 pr https://github.com/docker/compose-cli/pull/1830
```

Comment:
Would this flag be passed into the `.env`? Would users be aware that this flag exists?

Reply:
This isn't environmentally configurable. You can only control this via `./sandbox`'es arg `-v`

Reply:
Oops, sounds good - resolving